src/components/BusinessView.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState} from "react"
+import { SafeAreaView, View, FlatList, Text, Image, StyleSheet, TouchableOpacity, ScrollView } from "react-native"
+import Business from './Business'
+
+const ExampleBusiness: Business = {
+  id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28ba',
+  businessName: 'Starbucks Coffee',
+  distance: '1.2',
+  futureProofRating: 75,
+  userRating: 62,
+  logoURL: '../../assets/icon.png'
+}
+
+const ExampleBusinessDescription = "Starbucks Corporation is an American multinational chain of coffeehouses and roastery reserves headquartered in Seattle, Washington. It is the world's largest coffeehouse chain."
+
+const TitleView = ({business}: {business: Business}) => (
```

Comment:
It would be good to refactor this to its own file and import it. I understand the logic of wanting it to be close to the stuff, but I think the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) will serve us better long term.